packages/ui-extensions/src/surfaces/checkout/components/QRCode/QRCode.doc.ts

```diff
@@ -0,0 +1,66 @@
+import type {ReferenceEntityTemplateSchema} from '@shopify/generate-docs';
+
+import {getExample} from '../../helper.docs';
+
+const data: ReferenceEntityTemplateSchema = {
+  name: 'QRCode',
+  description: 'Used to quickly access scannable data.',
+  requires: '',
+  isVisualComponent: true,
+  thumbnail: 'qrcode-thumbnail.png',
+  type: '',
+  definitions: [
+    {
+      title: 'QRCodeProps',
+      description: '',
+      type: 'QRCodeProps',
+    },
+  ],
+  category: 'Components',
+  subCategory: 'Other',
+  defaultExample: {
+    image: 'qrcode-example.png',
+    codeblock: {
+      title: 'Basic QR Code',
+      tabs: [
+        {
+          title: 'React',
+          code: '../../../../../../ui-extensions-react/src/surfaces/checkout/components/QRCode/examples/basic-qrcode.example.tsx',
+          language: 'tsx',
+        },
+        {
+          title: 'JS',
+          code: './examples/basic-qrcode.example.ts',
+          language: 'js',
+        },
+      ],
+    },
+  },
+  subSections: [
+    {
+      type: 'Generic',
+      anchorLink: 'best-practices',
+      title: 'Best Practices',
+      sectionContent: `
```

Comment:
I believe we can write this as markdown right now, [see](https://github.com/Shopify/ui-extensions/pull/2014/files#diff-c068355d6a213f4ba727caca947dc925244c28050b713840f5b365cfb70df0a7R45-R64)

so no need for all of the \n\n

We can write and issue to go back and fix all other places that still utilizes this syntax.